src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java

```diff
@@ -333,7 +334,7 @@ private MGRSComponents breakMGRSString(String MGRSString)
                     error_code |= MGRS_STRING_ERROR;
             }
             else
-                error_code |= MGRS_STRING_ERROR;
+                zone = 0;
```

Comment:
A bit of a hack. But I don't see that zone can be 0. The whole point is so that there is no error_code, which will ensure `MGRSComponents` gets constructed, which then proceeds with conversion.